.github/comment-commands/retest.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#doc: add new empty commit to trigger new CI build
+
+PR_URL=$(jq -r '.issue.pull_request.url' "$GITHUB_EVENT_PATH")
+read -r REPO_URL BRANCH <<<"$(curl "$PR_URL" | jq -r '.head.repo.ssh_url + " " + .head.ref')"
```

Comment:
I tried `/retest` in the PR in your repo, but it [failed](https://github.com/elek/hadoop-ozone/pull/9#issuecomment-584615406).  Just guessing: maybe `-s` parameter for `curl` is missing and the extra output trips up `jq`?

Reply:
Seems to be a permission problem. For some reason github couldn't push. I can try to use https remote and the token as the password...

Reply:
OK, it's fixed now. Tested on the same PR